src/render.js

```diff
@@ -61,6 +61,7 @@ function cleanup() {
 
 function cleanupAtWrapper(wrapper) {
   if (
+    wrapper.element &&
     wrapper.element.parentNode &&
     wrapper.element.parentNode.parentNode === document.body
```

Comment:
I guess this is quite old code, it could be clarified:

`wrapper.element?.parentNode?.parentNode === document.body`